### ISSUE TITLE
Analyses: only Lighting runs

### ISSUE THREAD

**Issue by JeroneGagliano** [VOLTTRON/openeis#44]
I just installed the software from github using the Git program in Windows 7 and followed the Install instructions and the bootstrap.py was successful. I ran the activate openeis command, created a superuser account, then syncdb, runserver command, and then opened the localhost window in Chrome.  

I have created a datamap using the supplied sample data from the Guides folder and then selected the corresponding datasets UserGuide_lighting, UserGuide_WholeBuilding, and UserGuide_AHU to associate with the datamap.  However, when I click "Run Analysis..." and then select the dataset, the only application that runs is the Sensor Suitcase: Lighting. The others do not even show up.  When I look in the C:\openeis\openeis\applications folder, I see all of the analysis applications there. Why are they not running in the GUI?

I just ran (openeis)openeis runapplication in the cmd window to see what would happen and it gives the same Error: root:Module xxx cannot be imported. Reason: No module named 'numpy'.  It gives this error for all applications except Lighting which is the only one that runs inside of the OpenEIS GUI.

Thanks
Jerone

**Comment by hlngo** [comment]
@JeroneGagliano Installing numpy/scipy/pandas on Windows is problematic. To install numpy:
- Go to your openeis git folder
- Activate the environement ". env/bin/activate"
- Install numpy "pip install numpy"

Another way to use openeis on Windows is to use our installer.

**Comment by JeroneGagliano** [comment]
Yes, I was able to get it running once I used Anaconda.

My issue is that I cannot get any of analyses to run except for the
Lighting module.



Jerone Gagliano  PE, CEM, CMVP
607-592-4948

**Comment by hlngo** [comment]
@JeroneGagliano It's hard to help without more information. My best guess is that your data doesn't have required data points for most of the apps in OpenEIS. What data points are available in your dataset?

**Comment by JeroneGagliano** [comment]
I used the full hvac sample data set provided and mapped all points. From
my original post:

"I have created a datamap using the supplied sample data from the Guides
folder and then selected the corresponding datasets UserGuide_lighting,
UserGuide_WholeBuilding, and UserGuide_AHU to associate with the datamap.
However, when I click "Run Analysis..." and then select the dataset, the
only application that runs is the Sensor Suitcase: Lighting. The others do
not even show up. When I look in the C:\openeis\openeis\applications
folder, I see all of the analysis applications there. Why are they not
running in the GUI?"

Thanks
Jerone

Jerone Gagliano PE, CEM, CMVP
(607) 592-4948

**Comment by hlngo** [comment]
@JeroneGagliano I see. If the apps don't show up, that means you don't have some libraries in your VIRTUAL ENVIRONMENT. OpenEIS uses a separate python other than the one you installed using ananconda. 
From your description, you should have at least 2 environments on your system: 1) Anaconda; 2) OpenEIS virtual environment. 

Can you try this:
- Open Command Prompt / Terminal
- "cd <your OpenEIS folder>" (eg. "cd C:\OpenEIS") 
- Type ". env/bin/activate"
- Type "pip install numpy"

Then rerun OpenEIS and see if other apps are  there.

**Comment by JeroneGagliano** [comment]
Yes, that took care of it. I had to activate it using env/Scripts/active
and that put me in the (openeis) environment. I then did the pip install
numpy, which put numpy in the openeis environment. Before I was just typing
"active" which did something but did not change the command prompt head and
still let me run "openeis runserver".

Thanks
Jerone



Jerone Gagliano  PE, CEM, CMVP
607-592-4948

**Comment by hlngo** [comment]
@JeroneGagliano Glad it helped. You can use the same approach to install missing libraries. You can see missing libraries in the terminal when you first start openeis.
